jenkins/ubuntu1604_c.sh

```diff
@@ -12,6 +12,6 @@ build_root=$(cd "$(dirname "$0")/.." && pwd)
 cd $build_root
 
 # -- C --
-./build_all/linux/build.sh --run-unittests --run_valgrind --run-e2e-tests --run-sfc-tests --provisioning --use-edge-modules "$@" #-x 
```

Comment:
>--run-sfc-tests [](start = 74, length = 15)

Why is this one removed?

Reply:
Here and all the files.

---
In reply to: [313150340](https://github.com/Azure/azure-iot-sdk-c/pull/1107#discussion_r313150340) [](ancestors = 313150340)